Explorer/ProjectSettings/ProjectSettings.asset

```diff
@@ -231,7 +231,8 @@ PlayerSettings:
   iOSLaunchScreeniPadCustomStoryboardPath: 
   iOSDeviceRequirements: []
   iOSURLSchemes: []
-  macOSURLSchemes: []
+  macOSURLSchemes:
+  - decentraland
```

Comment:
sorry for the ignorance, what is this?

Reply:
Hey @davidejensen!, that's the automagic deep linking that Unity provides for some platforms, in our case it's only used for MacOS: https://docs.unity3d.com/Manual/deep-linking-macos.html

Surprisingly enough Windows Standalone (not Universal Windows builds) doesn't allow that and we rely now on a little batch script for registering the application in the OS registry for deep linking. Manually executed until the new Launcher does that for us.

But for MacOS we can just set it there, so that it works like: `open decentraland://` for opening it from a terminal or even calling it from a browser like `decentraland://`